src/breadcrumb/breadcrumb-item.component.ts

```diff
@@ -0,0 +1,23 @@
+import {
+	Component,
+	HostBinding,
+	Input
+} from "@angular/core";
+
+@Component({
+	selector: "ibm-breadcrumb-item",
+	template: `
+	<a class="bx--link"
+		href="{{href}}"
+		*ngIf="href; else content">
+		<ng-container *ngTemplateOutlet="content"></ng-container>
+	</a>
+	<ng-template #content>
+		<ng-content></ng-content>
+	</ng-template>`
+})
+export class BreadcrumbItemComponent {
```

Comment:
`Component` in the name here is fine since there is also a `BreadcrumbItem` interface which is more likely to be used more often than the component 👍